### DISCUSSION TITLE
Face swap with Base model + Refiner model dont work?

### DISCUSSION THREAD

**Discussion by Padraix** [lllyasviel/Fooocus#3118] [Q&A] [answered]
I try use face swap option and when i use base model (Juggernaut), its work, slowly but work. But its really slow, so i trying using Face swap with base model + refiner model , and everything going good, but when Fooocus start using refiner model (SD 1.5 model) , i have a random face generated.

**Comment by mashb1t** [comment, marked as answer]
The controlnet / IP adapter has been concepted for SDXL and does not convey its effects to SD1.5 in refiner. Same for all other modes.

**Reply by Padraix** [reply]
Ok, thanks.
